src/main/java/org/elasticsearch/search/aggregations/metrics/stats/extended/InternalExtendedStats.java

```diff
@@ -85,6 +90,9 @@ public double value(String name) {
         if ("std_deviation".equals(name)) {
             return getStdDeviation();
         }
+        if ("std_error".equals(name)) {
+            return getStdError();
+        }
```

Comment:
will need to add `std_upper`, `std_lower`, `confidence_upper`, `confidence_lower` here otherwise they won't be able to be used in sorting or `getProperty()`